lib/command/gherkin/snippets.js

```diff
@@ -34,77 +36,81 @@ module.exports = function (genPath, options) {
       process.exit(1);
     }
 
-    const files = [];
-    glob.sync(config.gherkin.features, { cwd: global.codecept_dir }).forEach((file) => {
-      if (!fsPath.isAbsolute(file)) {
-        file = fsPath.join(global.codecept_dir, file);
-      }
-      files.push(fsPath.resolve(file));
-    });
-    output.print(`Loaded ${files.length} files`);
+    codecept.runBootstrap((err) => {
```

Comment:
gherkin:snippets should not execute bootstrap (as it's omitted for def/list commands as well)